docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
       PHP_EXTENSION_PGSQL: 1
       PHP_EXTENSION_PDO_PGSQL: 1
       PHP_EXTENSION_MYSQLI: 1
-      STARTUP_COMMAND_1: composer install
+      STARTUP_COMMAND_1: composer install && composer set-config -- config/local.neon --database.driver=pdo_pgsql --database.host=postgres --database.dbname=apitte --database.user=apitte --database.password=apitte --database.port=5432
```

Comment:
Proc tahle zmena?

Reply:
jak bys to udělal, když je jiný host v doclkeru a v GA?

Reply:
aha, ono to funguje i bez toho

Reply:
to nepůjde přes sqlite3, není v tom PHP imagi

Reply:
Ikdyž jsem ho tam přidal, nejede to, musely by vytvořit migrace pro sqlit3.  Já bych celé sqlite zahodil.

Reply:
Ten prikaz nech jak byl prosim.